template.yaml

```diff
@@ -1016,7 +1016,7 @@ Resources:
       Runtime: java11
       Timeout: 30 #  30 sec timeout
       Role: !GetAtt LambdaRole.Arn
-      ReservedConcurrentExecutions: 64
+      ReservedConcurrentExecutions: 5
```

Comment:
Set the parallelism of the lambdas low for testing purposes and for avoiding overwhelming the EventBridge Bus.